tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -0,0 +1,263 @@
+                    EnvironmentInfo.SetVariable(variableName, variableValue);
+                    AzurePipelines.Instance.SetVariable(variableName, variableValue);
+
+                    return isChanged;
+                }
+            }
+
+            void GenerateIntegrationTestsWindowsMatrices(GenerateMatricesTarget target)
+            {
+                var targetFrameworks = TargetFramework.GetFrameworks(new[] { TargetFramework.NETSTANDARD2_0 });
+
+                if (target.HasFlag(global::GenerateMatricesTarget.integration_tests_windows_matrix))
+                {
+                    GenerateIntegrationTestsWindowsMatrix(targetFrameworks);
+                }
+
+                if (target.HasFlag(global::GenerateMatricesTarget.integration_tests_windows_iis_matrix))
+                {
+                    GenerateIntegrationTestsWindowsIISMatrix(targetFrameworks);
+                }
+            }
+
+            void GenerateIntegrationTestsWindowsMatrix(TargetFramework[] targetFrameworks)
+            {
+                var targetPlatforms = new[] { "x86", "x64" };
+                var matrix = new Dictionary<string, object>();
+
+                foreach (var framework in targetFrameworks)
+                {
+                    foreach (var targetPlatform in targetPlatforms)
+                    {
+                        matrix.Add($"{targetPlatform}_{framework}", new { framework, targetPlatform });
+                    }
+                }
+
+                Logger.Info($"Integration test windows matrix");
+                Logger.Info(JsonConvert.SerializeObject(matrix, Formatting.Indented));
+                AzurePipelines.Instance.SetVariable("integration_tests_windows_matrix", JsonConvert.SerializeObject(matrix, Formatting.None));
+            }
+
+            void GenerateIntegrationTestsWindowsIISMatrix(TargetFramework[] targetFrameworks)
+            {
+                var targetPlatforms = new[] { "x86", "x64" };
+
+                var matrix = new Dictionary<string, object>();
+                foreach (var framework in targetFrameworks)
+                {
+                    foreach (var targetPlatform in targetPlatforms)
+                    {
+                        var enable32bit = targetPlatform == "x86";
+                        matrix.Add($"{targetPlatform}_{framework}", new { framework, targetPlatform, enable32bit });
+                    }
+                }
+
+                Logger.Info($"Integration test windows IIS matrix");
+                Logger.Info(JsonConvert.SerializeObject(matrix, Formatting.Indented));
+                AzurePipelines.Instance.SetVariable("integration_tests_windows_iis_matrix", JsonConvert.SerializeObject(matrix, Formatting.None));
+            }
+
+            void GenerateIntegrationTestsLinuxMatrix()
+            {
+                var targetFrameworks = TargetFramework.GetFrameworks(new[] { TargetFramework.NET461, TargetFramework.NETSTANDARD2_0, });
```

Comment:
nit: this does the opposite of what I expected it to!
```suggestion
                var targetFrameworks = TargetFramework.GetFrameworks(except: new[] { TargetFramework.NET461, TargetFramework.NETSTANDARD2_0, });
```

Reply:
Done in d93ff92a0d67ffbad70bdaf16e461893405a8990